pkg/odo/genericclioptions/context.go

```diff
@@ -327,6 +327,10 @@ func NewOfflineDevfileContext(command *cobra.Command) *Context {
 		// this is only so we can make devfile and s2i work together for certain cases
 		LocalConfigInfo: &config.LocalConfigInfo{},
 	}
+	client, err := Client()
+	if err == nil {
+		internalCxt.Client = client
+	}
```

Comment:
We shouldn't initialize the client in `NewOfflineDevfileContext()`.

Reply:
How about now?